packages/block-library/src/index.js

```diff
@@ -148,6 +149,7 @@ export const __experimentalGetCoreBlocks = () => [
 	missing,
 	more,
 	nextpage,
+	pageList,
```

Comment:
So this block is stable and undependent of the navigation block, which means it's going to be included in the next WP major (5.8) regardless of the status of the Navigation block, I just want to make sure this is a conscious decision.

Reply:
@tellthemachines was the block meant to be tweaked with regards to syntax, or is it pretty much baked as is?

Reply:
It seems we should keep this tied to the navigation block (and its child blocks) for a bit

Reply:
In making this an independent block, I was thinking of the Widgets editor, as we don't currently have another block that can reproduce the Pages widget functionality. Assuming we want the Widgets editor to have feature parity with the legacy widgets screen, this block will be needed. 

I'm happy with its current syntax, and the fact that it gets whatever styling properties from Navigation block context means that when used standalone, it works just like the Categories or the Archives block, that don't have any styling options.

Related question: should Categories also work as a child of Navigation? There's definitely a use case for categories in menus.